app/styles/app.css

```diff
@@ -1,3 +1,24 @@
 article.post-content .float-right {
   float: right;
 }
+
+table {
+  table-layout: auto;
+  width: 100%;
+  border-collapse: collapse;
+  margin-bottom: var(--spacing-2);
+}
+
+th, td {
+  padding-left: var(--spacing-2);
+  padding-right: var(--spacing-2);
+}
+
+thead {
+  background-color: var(--color-gray-300);
+  border-bottom: 1px solid;
```

Comment:
```suggestion
  border-bottom: 1px solid;
  text-align: left;
```

My preference would be to align the `thead` left ;) but it's a preference thing
<img width="783" alt="Screenshot 2022-09-28 at 10 37 11" src="https://user-images.githubusercontent.com/5811560/192731548-2b012bb6-50ff-46b5-9086-1c0ef3e18b55.png">